vcls/pushmode/README.md

```diff
@@ -0,0 +1,37 @@
+# Clustered push-based objectstore
+
+This is an example with live video pushed to the objectstore
+
+# How to test:
+
+Push and fetch files:
+
+1. `docker compose up`
+   * starts 3 varnishes, and a loadbalancing varnish (just for reaching the varnishes)
+2. `curl http://localhost:6081/snake`
+   * get 404
+3. `curl -X POST --data-binary @../snake.six -H 'Content-Type: image/sixel' -H 'Authorization: secret' -v localhost:6081/snake`
```

Comment:
maybe push the `README.md` or `/etc/hostname` rather than a file the user won't have on their machine?

Reply:
Or just push the string "Hello world"?